### PR TITLE
adding 5 Chinese documents for Python.

### PR DESCRIPTION
**Pull request by bukun**
## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books-zh.md [hunk 1]
```diff
@@ -640,12 +640,17 @@
...
 * [Django book 2.0](http://djangobook.py3k.cn/2.0/)
+* [Matplotlib 3.0.3 中文文档](http://www.osgeo.cn/matplotlib/) - (Online)
+* [Numpy 1.16 中文文档](http://www.osgeo.cn/numpy/) - (Online)
 * [Python 3 文档(简体中文) 3.2.2 documentation](http://docspy3zh.readthedocs.org/en/latest/)
+* [Python 3.8.0a3中文文档](http://www.osgeo.cn/cpython/) - (目前在线最全的中文文档了，Online)
 * [Python Cookbook第三版](http://python3-cookbook.readthedocs.io/zh_CN/latest/) - David Beazley、Brian K.Jones、熊能(翻译)
 * [Python 中文学习大本营](http://www.pythondoc.com)
 * [Python之旅](http://funhacks.net/explore-python) - Ethan
 * [Python教程 - 廖雪峰的官方网站](http://www.liaoxuefeng.com/wiki/0014316089557264a6b348958f449949df42a6d3a2e542c000)
+* [Tonado 6.1 中文文档](http://www.osgeo.cn/tornado/) - (网络上其他的都是较旧版本的，Online)
```
Comment (eshellman) [inline free-programming-books-zh.md]: spelling?

File: free-programming-books-zh.md [hunk 2]
```diff
@@ -640,12 +640,17 @@
 ### Python
 
 * [Django 1.11.6 中文文档](https://www.yiyibooks.cn/xx/Django_1.11.6/index.html)
+* [Django 2.2.1 中文文档](http://www.osgeo.cn/django/) - (这个很新，也很全，Online)
```
Comment (zhongjiajie) [inline free-programming-books-zh.md]: I think we should remove describe. we just need author and type after the line.
```suggestion
* [Django 2.2.1 中文文档](http://www.osgeo.cn/django/)
```
so as other resource.

**Comment by eshellman** [comment]
Welcome to free-programming-books!

**Comment by zhongjiajie** [comment]
Er....merged.

**Pull request merged**